File: Serene/Serene.Web/Modules/Imports/ServiceTypings/Northwind.CustomerDemographicRow.ts
namespace Serene.Northwind
{
    export interface CustomerDemographicRow
    {
        ID: number;
        CustomerTypeID: string;
        CustomerDesc: string;
    }

    export namespace CustomerDemographicRow
    {
        export const IdProperty = "ID";
        export const NameProperty = "CustomerTypeID";
        export const LocalTextPrefix = "Northwind.CustomerDemographic";

        export namespace Fields
        {
            export declare const ID: "ID";
            export declare const CustomerTypeID: "CustomerTypeID";
            export declare const CustomerDesc: "CustomerDesc";
        }

        ["ID","CustomerTypeID","CustomerDesc"].forEach(x => (<any>Fields)[x] = x);
    }
}

File: Serene/Serene.Script/Imports/ServerImports/Northwind.CustomerDemographicRow.cs
using jQueryApi;
using Serenity;
using Serenity.ComponentModel;
using System;
using System.Collections;
using System.Collections.Generic;
using System.ComponentModel;
using System.Runtime.CompilerServices;

namespace Serene.Northwind
{
    [Imported, Serializable, PreserveMemberCase]
    public partial class CustomerDemographicRow
    {
        [InlineConstant] public const string IdProperty = "ID";
        [InlineConstant] public const string NameProperty = "CustomerTypeID";
        [InlineConstant] public const string LocalTextPrefix = "Northwind.CustomerDemographic";

        public Int32? ID { get; set; }
        public String CustomerTypeID { get; set; }
        public String CustomerDesc { get; set; }

        [Imported, PreserveMemberCase]
        public static class Fields
        {
            [InlineConstant] public const string ID = "ID";
            [InlineConstant] public const string CustomerTypeID = "CustomerTypeID";
            [InlineConstant] public const string CustomerDesc = "CustomerDesc";
        }
    }
}

